lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -52,19 +52,12 @@ class ChapterSerializer(serializers.Serializer):
 
 
 class CertificateDataSerializer(serializers.Serializer):
+    cert_status = serializers.CharField()
     cert_web_view_url = serializers.CharField()
     download_url = serializers.CharField()
-    is_downloadable = serializers.SerializerMethodField()
-    is_requestable = serializers.SerializerMethodField()
```

Comment:
Yes, this is bad. But this API is completely unused at the moment so I feel safe making this change.